src/cmr/concepts/__tests__/draft.test.js

```diff
@@ -70,18 +77,36 @@ describe('Draft concept', () => {
       describe('when the conceptType is variable-drafts', () => {
         test('sets the ingestPath and metadataSpecification correctly', () => {
           const draft = new Draft('variable-drafts', {}, {}, {
-            providerId: 'EDSC',
-            ummVersion: '1.0.0'
+            draftConceptId: 'VD100000-EDSC',
+            providerId: 'EDSC'
           })
 
           expect(draft.ingestPath).toEqual('providers/EDSC/variable-drafts')
+          expect(draft.publishPath).toEqual('publish/VD100000-EDSC')
           expect(draft.metadataSpecification).toEqual({
-            Name: 'variable-draft',
-            URL: 'https://cdn.earthdata.nasa.gov/umm/variable-draft/v1.0.0',
+            Name: 'UMM-Var',
+            URL: 'https://cdn.earthdata.nasa.gov/umm/variable/v1.0.0',
             Version: '1.0.0'
           })
         })
       })
+
+      describe('when the conceptType is not a supported draft', () => {
+        test('sets the ingestPath and metadataSpecification correctly', () => {
+          const draft = new Draft('bad-drafts', {}, {}, {
+            draftConceptId: 'CD100000-EDSC',
+            providerId: 'EDSC'
+          })
+
+          expect(draft.ingestPath).toEqual('providers/EDSC/bad-drafts')
+          expect(draft.publishPath).toEqual('publish/CD100000-EDSC')
+          expect(draft.metadataSpecification).toEqual({
```

Comment:
Seems like this should throw an error instead of return bad data... Is the idea here that we send this to CMR and let them report the issue?

Reply:
Its really unreachable code. The conceptType in that switch statement is based off a graphql enum type, but I didn't want to write a ton of if statements so I wrote a switch. eslint expects a default case so this test handles the default. I can change it to throw an error, but it is still something we wouldn't see

Reply:
Seeing all the tests I broke when I changed it to throw an error. When we call publishDraft it still uses this class, but conceptType is unimportant, so I just passed in `Draft`, which hits the default case in the switch. But the metadataSpecification isn't used when publishing a draft